.dockerignore

```diff
@@ -0,0 +1,47 @@
+# Just copy the .gitignore file for files we don't want to copy to the docker image
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+/dist
+
+# misc
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+.project
+.settings/
+.eslintcache
+.stylelintcache
+lerna-debug.log
+Lerna-Profile-*.json
+
+/public/vs
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+src/**/*.css
+
+tsconfig.tsbuildinfo
+packages/*/package-lock.json
+/test-results/
+/playwright-report/
+/playwright/.cache/
+
+# Ignore Dockerfile as well, since we don't need to copy that into the snapshot container
+Dockerfile
```

Comment:
Should also add `/tests` to the docker ignore. Those files are currently copied over, but if you only change a test file, the build should be fully cached

Reply:
Good call